Complex Radio/presence.ts

```diff
@@ -0,0 +1,26 @@
+var presence = new Presence({ clientId: "705189441484095508" });
+
+var ssong, slisteners, spresenter;
+
+async function newStats() {
+    const data = await window
+        .fetch("https://radio.complexr.pw/api/nowplaying/1")
+        .then((res) => res.json());
+    ssong = data.now_playing.song.text;
+    spresenter = data.live.is_live ? "Presenter " + data.live.streamer_name : "AutoDJ";
+    slisteners = data.listeners.unique + " Listeners";
+}
+
+setInterval(newStats, 2500);
+newStats();

```

Comment:
You are calling an async function without await. That's the problem.

Reply:
Right how would i fix this. As it seems to be fine when i load the presence

Reply:
It's part of the reason that you are getting the DeepScan issue.

Reply:
https://i.imgur.com/LevJzfA.png

Reply:
still dk how to fix that deepscan